src/monitors/rocketMonitor.cpp

```diff
@@ -19,8 +19,9 @@ ablate::monitors::RocketMonitor::RocketMonitor(const std::string nameIn, std::sh
 
 void ablate::monitors::RocketMonitor::Register(std::shared_ptr<solver::Solver> solver) {
     Monitor::Register(solver);
-    auto comm = GetSolver()->GetSubDomain().GetComm();                                                                               // The communicator in which the reduction takes place.
-    computePressure = eos->GetThermodynamicFunction(eos::ThermodynamicProperty::Pressure, GetSolver()->GetSubDomain().GetFields());  // get decode state function/context
+    auto comm = GetSolver()->GetSubDomain().GetComm();                                                                                       // The communicator in which the reduction takes place.
+    computePressure = eos->GetThermodynamicFunction(eos::ThermodynamicProperty::Pressure, GetSolver()->GetSubDomain().GetFields());          // get decode state function/context
```

Comment:
If we are calling both of these in the same spot, we probably want to get the temperature first and then use that temperature to compute pressure and speed of sound.